inbm/proto/common/v1/common.proto

```diff
@@ -67,22 +68,43 @@ message Operation {
 }
 
 message RpcActivateOperation {
-  string url = 1;
+  string url = 1 [(buf.validate.field).cel = {
+      id: "valid_uri"
+      message: "url must be a valid URL."
+      expression: "this.isUri()"
+    },
+    (buf.validate.field).required = true];;
   string profile_name = 2;
 }
 
 message UpdateSystemSoftwareOperation {
-    string url = 1; // URL from which to remotely retrieve the package
-    google.protobuf.Timestamp release_date = 2; // Release date of the new SW update.
+    string url = 1 [(buf.validate.field).cel = {
```

Comment:
Is it possible to run validate on these messages in cloudadapter? Or do we assume they're already validated higher in the stack, in INBS or MJunct?

Reply:
These messages will be validated in higher stack, which is MJunct. For example, https://github.com/intel-innersource/frameworks.edge.one-intel-edge.manageability.unified-device-management/blob/027b2b87eb2e616f6d44abf5e764f1f8aa078e5d/internal/mjunct/mjunct.go#L74